### PR TITLE
some naming issues when merged, didnt change some old names to match new...

### PR DESCRIPTION
**Pull request by gduck**
... columns and broke the search & category function. Fixed here

ready to merge into master

### CODE REVIEW

File: app/assets/templates/header.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
           </button>
           <ul class="dropdown-menu list-inline" role="menu">
             <li class="dropdown-list" ng-repeat='category in categories'>
-              <a ng-href="#{{category.link}}" class="dropdown-category">{{ category.name }}</a>
+              <a ng-href="#/categories/{{ category.link }}" class="dropdown-category">{{ category.name }}</a>
```
Comment (paulld) [inline app/assets/templates/header.html]: do you still use the link? I thought you changed to simplified_name? so your url would look like categories/{{category.simplified_name}}

Comment (gduck) [inline app/assets/templates/header.html]: You have keen eyes!! 
We kept the jbuilder the same to minimise the html coding changes. So the jbuilder is
json.link category.simplified_name

Comment (paulld) [inline app/assets/templates/header.html]: If you're sure that you won't get confused it's fine. however in the long term such discrepancies can lead to errors! How long would it take to standardise?

Comment (gduck) [inline app/assets/templates/header.html]: not long. can do

**Pull request merged**